src/System.Runtime.Analyzers/VisualBasic/System.Runtime.VisualBasic.Analyzers.vbproj

```diff
@@ -116,8 +116,8 @@
     <Compile Include="BasicRethrowToPreserveStackDetails.Fixer.vb" />
     <Compile Include="BasicDoNotRaiseReservedExceptionTypes.vb" />
     <Compile Include="BasicDoNotRaiseReservedExceptionTypes.Fixer.vb" />
-    <Compile Include="BasicInitializeValueTypeStaticFieldsInline.vb" />
-    <Compile Include="BasicInitializeValueTypeStaticFieldsInline.Fixer.vb" />
+    <Compile Include="BasicInitializeStaticFieldsInline.vb" />
```

Comment:
I don't think we want more than one unit test projects per analyzer - so we don't want a VB project for testing VB implementation of the analyzer + C# project for testing C# implementation. How about the following:

1. Retain Gen's current project structure - have just a single C# unit test project per analyzer.
2. Create a single [RuleName]Tests.cs file for all rule tests.
3. For analyzers that implement a single CA rule (and most analyzers will do so), we should name the tests without the language or CA rule prefix and put equivalent C# and VB tests in the same unit test - this is what Gen did for his PR (see [here](https://github.com/dotnet/roslyn-analyzers/pull/340/files#r44174198)), and I like that approach.
4. This analyzer merges 2 rules, so its probably okay for the tests for it to have CAXXXX prefix OR create separate nested types for each rule tests - its just a minor coding style preference which applies only for a minority of rules.